src/plugins/discover/public/application/context/context_app.tsx

```diff
@@ -36,11 +37,17 @@ export interface ContextAppProps {
 
 export const ContextApp = ({ indexPattern, anchorId }: ContextAppProps) => {
   const services = useDiscoverServices();
-  const { uiSettings, capabilities, indexPatterns, navigation, filterManager } = services;
+  const { uiSettings, capabilities, indexPatterns, navigation, filterManager, core } = services;
 
   const isLegacy = useMemo(() => uiSettings.get(DOC_TABLE_LEGACY), [uiSettings]);
   const useNewFieldsApi = useMemo(() => !uiSettings.get(SEARCH_FIELDS_FROM_SOURCE), [uiSettings]);
 
+  useExecutionContext(core.executionContext, {
+    type: 'application',
```

Comment:
Similar to my previous comment, according to `src/core/types/execution_context.ts`, the `type` field is used to report the app:
```ts
  /**
   * Kibana application initated an operation.
   * */
  readonly type?: string; // 'visualization' | 'actions' | 'server' | ..;
```

Probably I'm missing some context though.